commcare_connect/templates/reports/htmx_table.html

```diff
@@ -0,0 +1,87 @@
+{% extends "django_tables2/bootstrap5.html" %}
+
+{% load django_tables2 %}
+{% load i18n %}
+
+{% block table.thead %}
+  {% if table.show_header %}
+      <thead {{ table.attrs.thead.as_html }}>
+      <tr>
+          {% for column in table.columns %}
+              <th {{ column.attrs.th.as_html }}
+                  {% if column.orderable %}
+                  hx-get="{% querystring table.prefixed_order_by_field=column.order_by_alias.next%}&{{ table.prefixed_page_field }}=1"
+                  {% endif %}
+                  hx-trigger="click"
+                  hx-target="div.table-container"
+                  hx-swap="outerHTML"
+                  style="cursor: pointer;">
+                  {{ column.header }}
+                  {% if column.orderable %}
+                    {% if column.order_by_alias == column.order_by_alias.next %}
+                    <i class="bi bi-arrow-down-short"></i>
+                    {% elif column.order_by_alias|slice:":1" == "-" %}
+                    <i class="bi bi-arrow-down-square-fill"></i>
+                    {% else %}
+                    <i class="bi bi-arrow-up-square-fill"></i>
+                    {% endif %}
+                  {% endif%}
+              </th>
+          {% endfor %}
+      </tr>
+      </thead>
+  {% endif %}
+{% endblock table.thead %}
+
+{% block pagination %}
```

Comment:
Do we have pagination in this report? I was unable to find it in the view.

Reply:
Yeah, we don't have pagination for this report, but this template is basically the common template that can be used everywhere. Taken from https://github.com/dimagi/commcare-connect/pull/349 (if this were merged, then this would have just been a DRY reference)